quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliVersionIT.java

```diff
@@ -42,77 +31,4 @@ public void shouldVersionMatchQuarkusVersion() {
         // Using shortcut
         assertEquals("Client Version " + Version.getVersion(), cliClient.run("-v").getOutput());
     }
-
-    @Test
-    public void shouldCreateApplicationOnJvm() {
```

Comment:
I've just realised that these scenarios were duplicated. We should keep the ones in QuarkusCliCreateJvmApplicationIT